### ISSUE TITLE
Broken volume links

### ISSUE THREAD

**Issue by BNAndras** [mradamcox/loc-insurancemaps#33]
I found two broken volume URLs this morning.

Alexandria, Rapides Parish - 1885
https://oldinsurancemaps.net/loc/sanborn03267_001/

Natchitoches, Natchitoches Parish - 1909
https://oldinsurancemaps.net/loc/sanborn03374_004/

**Comment by BNAndras** [comment]
https://oldinsurancemaps.net/loc/sanborn03269_001/ similarly broke after I opened the first sheet to prepare it.

**Comment by BNAndras** [comment]
It appears documents within the broken volumes are broken as well. https://oldinsurancemaps.net/documents/1204

**Comment by BNAndras** [comment]
[Breaux Bridge, La. | 1899](https://oldinsurancemaps.net/loc/sanborn03284_001/) is also broken. It was working last night when I initially loaded the volume.

**Comment by mradamcox** [comment]
Moving this to the volume summary milestone, as it's likely that the underlying problems are caused by errors during the volume serialization, a process that should fail silently, it seems, and perhaps provide a message in the volume summary page.

**Comment by mradamcox** [comment]
Underlying issues here are either
1. Multiple FullThumbnails have been inadvertently created for a document
2. Multiple SplitEvaluations have been created for a document.

To fix 1, I ran some queries and found 9 documents in all that had multiple FullThumbnails (often 3, sometimes 2)
```
Natchitoches, La. | 1909 p2
Natchitoches, La. | 1909 p3 [3]
Natchitoches, La. | 1909 p4 [2]
Natchitoches, La. | 1909 p4 [3]
Natchitoches, La. | 1909 p4 [1]
Natchitoches, La. | 1909 p3 [1]
Alexandria, La. | 1885 p2 [2]
Natchitoches, La. | 1909 p4 [4]
Breaux Bridge, La. | 1899 p1 [2]
```
I deleted all of the extra thumbs for each one of these documents and everything loads properly now. FullThumbnail generation has been improved and these duplicates shouldn't happen in the future.

Similarly, for 2, I found the following documents with multiple SplitEvaluations:
```
Arcadia, La. | 1903 p2 (2 evals)
New Orleans, La. | 1887 | Vol. 3 p57L (2 evals)
Amite, La. | 1886 p1 (3 evals)
```
and just decided to delete all of those, so they are now back to "unprepared".

---

Both 1 and 2 will be handled better when the newest version is deployed.
